src/App.jsx

```diff
@@ -1,35 +1,91 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { nanoid } from 'nanoid'
 import './App.css'
+import TodoItem from './components/TodoItems'
```

Comment:
Should we import this as `TodoItems`?
TodoItem name seems like the component for single todo entry

Reply:
Or we can rename this to `TodoList`. That'll be more appropriate.